packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -55,3 +106,19 @@ class DeepLinksController {
     _updateLinks();
   }
 }
+
+class _DomainAndPath {
```

Comment:
instead of creating a Dart class for this, can we just use a Record?

Reply:
ah I wasn't aware record is already available in the dart version dev tool is using